resources/plugins/arty2/plugin.json

```diff
@@ -0,0 +1,38 @@
+{
+    "nameInUI": "Carsten's Arty Spotter for Single Player",
+    "defaultValue": false,
+    "specificOptions": [
+        {
+            "nameInUI": "Time to impcat of the rounds",
+            "mnemonic": "user_fireDelay",
+            "defaultValue": 15
+        },
+        {
+            "nameInUI": "Salvo quantity",
+            "mnemonic": "user_quantity",
+            "defaultValue": 5
+        },
+        {
+            "nameInUI": "Impact radius",
+            "mnemonic": "user_spread",
+            "defaultValue": 150
+        },
+        {
+            "nameInUI": "Max spotting distance. In kilometers.", 
+            "mnemonic": "user_spottingDistance",
+            "defaultValue": 15
+        }
+    ],
+    "scriptsWorkOrders": [
+        {
+            "file": "CG_ArtySpotter_V1_2a_MP.lua",
+            "mnemonic": "arty"
+        }
```

Comment:
Needs to be adjusted, i.e. the file-name should corresponds with the SP version and the mnemonic would be arty2 in this case.